### PR TITLE
[5.8] Database Connection URLs

### PR DESCRIPTION
**Pull request by ahinkle**
PR adds note about the `url` option within the connection configuration.

https://github.com/laravel/framework/pull/28308

### CODE REVIEW

File: database.md [hunk 1]
```diff
@@ -73,6 +73,10 @@ You only need to place items in the `read` and `write` arrays if you wish to ove
 
 The `sticky` option is an *optional* value that can be used to allow the immediate reading of records that have been written to the database during the current request cycle. If the `sticky` option is enabled and a "write" operation has been performed against the database during the current request cycle, any further "read" operations will use the "write" connection. This ensures that any data written during the request cycle can be immediately read back from the database during that same request. It is up to you to decide if this is the desired behavior for your application.
 
+#### The `url` Option
+
+Using Heroku or another "cloud" database provider, you may need to specify the database connection URL. The `url` option is an *optional* value that can be used to specify the URL of your database connection. You can configure the database connection URL by setting the `DATABASE_URL` value within your `.env` environment file.
+
```
Comment (mathieutu) [inline database.md]: Maybe should we add more explanations, and some examples?

```suggestion

Scheme:

    DATABASE_URL=driver://username:password@host:port/database?options
 
Examples: 

    DATABASE_URL=sqlite:///path/to/database.sqlite
    DATABASE_URL=sqlite:///:memory:
    DATABASE_URL=mysql://forge:@localhost/forge?charset=UTF-8

All the non null parts of the url will overwrite the standard config keys. 
As an example, this config:

   [
        'url' => 'mysql://root:pass@db/local',
        'driver' => 'mysql',
        'host' => '127.0.0.1',
        'port' => '3306',
        'database' => 'forge',
        'username' => 'forge',
        'password' => '',
        'charset' => 'utf8mb4',
        'collation' => 'utf8mb4_unicode_ci',
        'strict' => true,
        'options' => ['foo' => 'bar'],
    ]

Will be read as:

    [
        'driver' => 'mysql',
        'host' => 'db',
        'port' => '3306',
        'database' => 'local',
        'username' => 'root',
        'password' => 'pass',
        'charset' => 'utf8mb4',
        'collation' => 'utf8mb4_unicode_ci',
        'strict' => true,
        'options' => ['foo' => 'bar'],
    ]

```

File: database.md [hunk 2]
```diff
@@ -73,6 +73,10 @@ You only need to place items in the `read` and `write` arrays if you wish to ove
 
 The `sticky` option is an *optional* value that can be used to allow the immediate reading of records that have been written to the database during the current request cycle. If the `sticky` option is enabled and a "write" operation has been performed against the database during the current request cycle, any further "read" operations will use the "write" connection. This ensures that any data written during the request cycle can be immediately read back from the database during that same request. It is up to you to decide if this is the desired behavior for your application.
 
+#### The `url` Option
+
+Using Heroku or another "cloud" database provider, you may need to specify the database connection URL. The `url` option is an *optional* value that can be used to specify the URL of your database connection. You can configure the database connection URL by setting the `DATABASE_URL` value within your `.env` environment file.
```
Comment (mathieutu) [inline database.md]:
```suggestion
Using Heroku or another "cloud" database provider, you may need to specify the database connection URL. The `url` option is an *optional* value that can be used to specify the URL of your database connection. You can configure the database connection URL by setting the `DATABASE_URL` value within your `.env` environment file or in your system environment.
```

**Comment by taylorotwell** [comment]
I don't think this makes a lot of sense nested under "Read / Write" connections since it has nothing to do with that. Also I think an example as suggested could be useful.

**Comment by taylorotwell** [comment]
I massaged it a bit and moved it.

**Pull request merged**